smart_sa/templates/intervention/activity.html

```diff
@@ -28,7 +28,7 @@
 
 {% block breadcrumb-main %}
 <div id="breadcrumb-main">
-	<a href="/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">Home</a> &rarr; Counsel &rarr; Working with <b>{{participant.display_name}}</b> 
+    <a href="/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">Home</a> &rarr; <a href="/intervention/{{activity.clientsession.intervention.id}}/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">Sessions</a> &rarr; <a href="/session/{{activity.clientsession.id}}/" onclick="if(document.completeactivity){document.completeactivity.next.value=this.href;document.completeactivity.submit();return false}">{{activity.clientsession.long_title}}</a> 
```

Comment:
As the click handlers are duplicated, now would be a good time to set up a common click function to handle these navigations.

Reply:
Got it, I'll work on that.